aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<p>Make sure to remove your own ARN and account ID as well. I recommend something like:</p><p><code>model_package_arn =&nbsp;"&lt;your-model-package-ARN&gt;"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>This ARN is from the model package hosted in AWS Marketplace. It will be the same ARN for everyone who subscribes to the model package.</p>

Reply:
<p>I see the ARN specifies <code>us-east-2</code>  - can this ARN be used from any region? Or does it need to be modified to contain the region in which a customer runs the example? I.e., if a customer runs this example in <code>us-west-2</code>,  does the ARN change?</p>

Reply:
Yes. The ARN changes based on the region. I have called out the step from where the ARN needs to be copied (it's just before the cell with this code) - 

> Accept Offer" button needs to be clicked if your organization agrees with EULA, pricing information as well as support terms. If Continue to configuration button is active, it means your account already has a subscription to this listing. Once you click on Continue to configuration button and then choose region, you will see that a Product ARN will appear. Specify the value in the following cell.